src/lib/ui/thermostat/TemperatureControl.stories.tsx

```diff
@@ -0,0 +1,75 @@
+import { Box } from '@mui/material'
+import type { Meta, StoryObj } from '@storybook/react'
+import { useState } from 'react'
+
+import { TemperatureControl } from './TemperatureControl.js'
+
+const meta: Meta<typeof TemperatureControl> = {
+  title: 'Library/TemperatureControl',
+  tags: ['autodocs'],
+  component: TemperatureControl,
+}
+
+type Story = StoryObj<typeof TemperatureControl>
+
+export const Content: Story = {
```

Comment:
I wonder if these should be separate stories? They would each get a name then and we wouldn't need any of the layout stuff. I think stories work best when they map one story to one component.

Reply:
Is this largely in contrast to the current convention? Taking a look at the `Button.stories.tsx`, `ClimateSettingStatus.stories.tsx` files, etc..

Happy to add additional stories, it would just be different than the current approach for atomic components. I kind of like the current approach where larger components have different stories for different context, but atomic components use one story for minor variations in props.

Reply:
Ok. Main feedback was when I looked at this story I didn't understand what I was looking at (I first I expected the sliders to somehow interact). [This story](https://react.seam.co/?path=/docs/library-climatesettingstatus--docs) doesn't have that issue for the most part maybe because there is no interaction.

Headings might help here. Not a big deal either way for the ui stories. They should be useful for dev purposes, so really whatever makes sense for us. One consideration is breaking up the stories will isolate the ui regression tests, so if there is an issue w/ one slider it will be clear which variant changed since the story will have a name.

Reply:
Ahh I see, okay. Maybe we can address this on a larger scale in another PR perhaps? I like the idea of component labels for stories—but also different stories is alright as well.